e2e/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
+/playwright/.cache/
+/playwright-report/
+/playwright/.cache/
+/e2e-reports/
```

Comment:
I think it won't be a problem to remove this `gitignore` file (it was auto-generated) and to move on the top level all these folders. 
Thanks for the suggestion, I will do it.

Reply:
Regarding the `package.json` the situation is more complex. I'm definitely not an expert on such configuration topics, but I discuss that with @kachar and he agreed that it's better to separate all `playwright` packages into the E2E folder. 
In that case will be easier to upgrade the versions, because currently I'm hitting some issues/errors when trying to do it on the main `package.json`.

Reply:
As I said, I'm not an expert on this, so I will be grateful to have good discussion about Pros and Cons of such approach. 
Thanks overall for your comments, @dimitur2204

Reply:
I see no problem in having these folders ignored in the current folder `.gitignore` file